config/nginx.conf

```diff
@@ -9,13 +9,13 @@ server {
      location /forum/ { }
    
      location = / {
-	rewrite ^/(.*)$ /forum/$1 redirect;
+	     rewrite ^/(.*)$ /forum/$1 redirect;
```

Comment:
I noticed when I tried to visit `http://127.0.0.1:2404` this redirect would get rid of the port and redirect me to `http://127.0.0.1/forum/`, which doesn't load

Reply:
What do you have `SITE_URL` envvar set to?
I use `http://wetfishonline.com.local:2404/forum` for that and in my /etc/hosts and it works okay.

However when I hit just http://wetfishonline.com.local:2404 it redirects to
https://wetfishonline.com.local/forum
which is kinda interesting

Reply:
Addressed in [cad03c6](https://github.com/wetfish/online/pull/135/commits/cad03c6d8257219110160c3b2792ea3dd7d4c3c7)